// @opentelemetry/configuration/build/src/models/commonModel.js
/*
 * Copyright The OpenTelemetry Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
'use strict';
Object.defineProperty(exports, "__esModule", { value: true });
exports.SeverityNumber = exports.OtlpHttpEncoding = void 0;
var OtlpHttpEncoding;
(function (OtlpHttpEncoding) {
    OtlpHttpEncoding["JSON"] = "json";
    OtlpHttpEncoding["Protobuf"] = "protobuf";
})(OtlpHttpEncoding = exports.OtlpHttpEncoding || (exports.OtlpHttpEncoding = {}));
var SeverityNumber;
(function (SeverityNumber) {
    SeverityNumber["DEBUG"] = "debug";
    SeverityNumber["DEBUG2"] = "debug2";
    SeverityNumber["DEBUG3"] = "debug3";
    SeverityNumber["DEBUG4"] = "debug4";
    SeverityNumber["ERROR"] = "error";
    SeverityNumber["ERROR2"] = "error2";
    SeverityNumber["ERROR3"] = "error3";
    SeverityNumber["ERROR4"] = "error4";
    SeverityNumber["FATAL"] = "fatal";
    SeverityNumber["FATAL2"] = "fatal2";
    SeverityNumber["FATAL3"] = "fatal3";
    SeverityNumber["FATAL4"] = "fatal4";
    SeverityNumber["INFO"] = "info";
    SeverityNumber["INFO2"] = "info2";
    SeverityNumber["INFO3"] = "info3";
    SeverityNumber["INFO4"] = "info4";
    SeverityNumber["TRACE"] = "trace";
    SeverityNumber["TRACE2"] = "trace2";
    SeverityNumber["TRACE3"] = "trace3";
    SeverityNumber["TRACE4"] = "trace4";
    SeverityNumber["WARN"] = "warn";
    SeverityNumber["WARN2"] = "warn2";
    SeverityNumber["WARN3"] = "warn3";
    SeverityNumber["WARN4"] = "warn4";
})(SeverityNumber = exports.SeverityNumber || (exports.SeverityNumber = {}));
